deploy/prod/remote-scripts/undeploy.sh

```
@@ -1,4 +1,4 @@
 #!/bin/bash
-
-cd /root/saf-2025/deploy/instance/remote-assets
+sudo -i
+cd /root/saf-2025/deploy/prod/remote-assets
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)